javascript/features/communication_commands/communication_commands.js

```diff
@@ -11,7 +11,7 @@ import { MuteCommands } from 'features/communication_commands/mute_commands.js'
 
 // Set of `/show` messages that Gunther will issue at a particular interval.
 const kGuntherMessages =
-    ['beg', 'discord', 'donate', 'forum', 'irc', 'reg', 'report', 'rules', 'weapons'];
+    ['beg', 'derby', 'discord', 'donate', 'forum', 'irc','minigames', 'reg', 'report', 'rules','stunt', 'weapons'];
```

Comment:
Please add a before `'minigames'` and `'stunt`', and make sure that the line is no longer than a hundred characters. If you use Visual Studio Code as your editor, a line will be shown automatically.